iroha/src/peer.rs

```diff
@@ -50,6 +53,20 @@ impl Peer {
                 .collect(),
             listen_address: id.address,
             domains,
+            #[cfg(feature = "bridge")]
+            bridges: Default::default(),
+        }
+    }
+
+    /// Constructor with filled `domains` field.
+    pub fn from_domains(id: PeerId, domains: HashMap<String, Domain>) -> Peer {
+        Peer {
+            id: id.clone(),
+            peers: Default::default(),
+            listen_address: id.address,
+            domains,
+            #[cfg(feature = "bridge")]
+            bridges: Default::default(),
```

Comment:
as I remember you can replace two default fields with `..Default::default()` in the end?

Reply:
@humb1t yeah, I forgot that it leads to unconditional recursion only with manually implemented `Default`. Fixed.